### DISCUSSION TITLE
Adding python packages to docker

### DISCUSSION THREAD

**Discussion by thsmit** [Kitware/trame#644] [Trame/ParaView] [answered]
Dear all,

I am building a trame app (vue2) and I am using 'non-standard' python packages. They are installed in the venv via pip. What is the right way to add these python packages to docker so the app will work via a docker container as well?

**Comment by jourdain** [comment, marked as answer]
You have access to a `requirements.txt` or `initialize.sh` to do anything you want.

**Reply by thsmit** [reply]
Thanks, using `initialize.sh` for now.
